src/System Application/App/Table Keys/src/TableKey.Codeunit.al

```diff
@@ -0,0 +1,39 @@
+namespace System.Reflection;
+
+/// <summary>
+/// Provides functionality for disabling and re-enabling table indexes.
+/// </summary>
+codeunit 9557 "Table Key"
+{
+    /// <summary>
+    /// Disables all keys on the provided table. Disabling keys before bulk table write operations can significantly improve performance.
+    /// </summary>
+    /// <param name="TableNo">The table to disable all of the keys for.</param>
+    /// <returns>True, if the keys were disabled successfully, false otherwise.</returns>
+    /// <remarks>
+    /// System tables and non-sql based tables are not supported for this operation.
+    /// Clustered keys, unique keys, SIFT keys, Nonclustered Columnstore Indexes are not affected by this operation.
+    /// The keys are automatically re-enabled when a Commit() is called, or at the end of AL code execution.
+    /// </remarks>
+    [Scope('OnPrem')]
```

Comment:
Is it planned to remove that later?

Reply:
Yes 🙂, but for the first iteration we want to limit the usage of this by 3rd party apps in SaaS